svc-events/events/publishevttosubscriber.go

```diff
@@ -68,114 +68,121 @@ func PublishEventsToDestination(data interface{}) bool {
 		log.Println("incoming event:", requestData)
 		return false
 	}
-	if message.Events[0].OriginOfCondition == nil {
-		log.Println("Event not forwarded : Originofcondition is empty in incoming event", requestData)
-		return false
-	}
 
-	if len(message.Events[0].OriginOfCondition.Oid) < 1 {
-		log.Println("Event not forwarded : Originofcondition is empty in incoming event", requestData)
-		return false
-	}
-	if strings.EqualFold(message.Events[0].EventType, "ResourceAdded") &&
-		strings.HasPrefix(message.Events[0].OriginOfCondition.Oid, "/redfish/v1/Fabrics") {
-		addFabricRPCCall(message.Events[0].OriginOfCondition.Oid, host)
-	}
+	var flag bool
+	eventMap := make(map[string][]common.Event)
+	for _, inEvent := range message.Events {
+		if inEvent.OriginOfCondition == nil {
+			log.Println("Event not forwarded : Originofcondition is empty in incoming event", requestData)
```

Comment:
Give space or any special character to separate the log text and the requestData